public/content/en/basics/basic-types.md

```diff
@@ -33,12 +33,12 @@ between floating point types (e.g `double` to `float`)
 is allowed though.
 
 A conversion to another type may be forced by using the
-`cast(TYPE) var` expression. It needs to be used with great care though
+`cast(TYPE) <var>` expression. It needs to be used with great care though
```

Comment:
I understood that. It's just that `<var>` is displayed which isn't valid D syntax and might be more confusing to the reader than the word `var`. What about just using `cast(TYPE) someVar` or  `cast(TYPE) myVariable`?